src/core/server/integration_tests/saved_objects/migrations/group3/actions/actions_test_suite.ts

```diff
@@ -1510,8 +1510,7 @@ export const runActionTestSuite = ({
     });
   });
 
-  // FLAKY: https://github.com/elastic/kibana/issues/176846
-  describe.skip('updateAndPickupMappings', () => {
+  describe('updateAndPickupMappings', () => {
```

Comment:
It seems that the backported PRs did not address the issue. CI keeps failing consistently, so we must be missing something.